File: Python/5thsem/module 1/calc.py
print("Enter two operators: ")
a=int(input())
b=int(input())
print("Enter operand: ")
op=input()

def func(a,b,op):
    if op=="+":
        return a+b
    elif op=='-':
        return a-b
    elif op=='*':
        return a*b
    elif op=='/':
        return a/b

print(func(a,b,op))

File: C/2nd Sem/Module 2/7. calculator/calc.c
#include<stdio.h>
int main()
{
    float a,b,c;
    printf("Enter two values:\n");
    scanf("%f %f",&a,&b);
    c=a+b;
    printf("Addition of two numbers =%f \n",c);
    c=a-b;
    printf("Subtraction of two numbers =%f \n",c);
    c=a*b;
    printf("Multiplication of two numbers =%f \n",c);
    c=a/b;
    printf("Division of two numbers =%f \n",c);

}